### PR TITLE
Put playmode icons on home beatmapset panels

### LINKED ISSUE #6017: Make new Ranked and Popular Beatmaps display mode
Every map here is taiko:
image
We need to see mode before clicking.

### PR DESCRIPTION
**Pull request by venix12**
resolves #6017

Went for this simple kind of design.

### CODE REVIEW

**Comment by nanaya** [comment]
hi @arflyte can you check this thanks

**Comment by arflyte** [comment]
Hmm, just to show the playmode? Yeah, we can go with this.

File: resources/assets/less/bem/user-home-beatmapset.less [hunk 1]
```diff
@@ -65,8 +65,20 @@
     color: @osu-colour-f1;
   }
 
+  &__playmode-icon {
+    font-size: @font-size--normal-2;
+
+    &:not(:first-of-type) {
+      margin-left: 2px;
+    }
```
Comment (nanaya) [inline resources/assets/less/bem/user-home-beatmapset.less:73]: this and the margin left on `__title` can be replaced with just margin right on `__playmode-icon`?

**Review by nanaya — DISMISSED** [review]

File: resources/views/home/_user_beatmapset.blade.php [hunk 2]
```diff
@@ -9,7 +9,15 @@
         srcSet="{{$beatmapset->allCoverURLs()['list']}} 1x, {{$beatmapset->allCoverURLs()['list@2x']}} 2x">
 
     <div class="user-home-beatmapset__meta">
-        <div class='user-home-beatmapset__title u-ellipsis-overflow'>{{$beatmapset->title}}</div>
+        <div class="user-home-beatmapset__title-container">
+            @foreach ($beatmapset->playmodesStr() as $playmode)
```
[the remark this answers is not in the document]
Comment (venix12) [reply resources/views/home/_user_beatmapset.blade.php:13]: how would you go about avoiding it?

[the remark this answers is not in the document]
Comment (nanaya) [reply resources/views/home/_user_beatmapset.blade.php:13]: preload on beatmapset query

**Comment by peppy** [comment]
@venix12 are you planning on applying the review changes for this?

**Review by notbakaneko — APPROVED** [review]

**Pull request merged**